### DISCUSSION TITLE
QTable - select at expand rows

### DISCUSSION THREAD

**Discussion by neluckoff** [quasarframework/quasar#16035] [General - Components / Directives / etc]
### Description

Hello! I want to make a table that will have some nesting in the selection of elements. You can see it on the screenshot.

![image](https://github.com/quasarframework/quasar/assets/80652352/6bd17beb-ce4a-4e6d-ba61-a9f8342f8dae)

But the problem is that I did not find how to connect more drop-down lines there. I hope such functionality exists.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar), Directives (quasar)

**Comment by neluckoff** [comment]
I'm trying to use the following string structure

```
rows: [
        {
          name: 'Договор 1234567890',
          childrens: [
            {
              calories: 159,
              fat: 6.0,
              carbs: 24,
              protein: 4.0,
              sodium: 87,
              calcium: '14%',
              iron: '1%',
            },
            {
              calories: 159,
              fat: 6.0,
              carbs: 24,
              protein: 4.0,
              sodium: 87,
              calcium: '14%',
              iron: '1%',
            }
          ]
        },
        {
          name: 'Договор 1234',
          childrens: [
            {
              calories: 159,
              fat: 6.0,
              carbs: 24,
              protein: 4.0,
              sodium: 87,
              calcium: '14%',
              iron: '1%',
            }
          ]
        }
      ]

**Comment by metalsadman** [comment]
try to put it in a codepen so peeps can make an edit.

**Reply by neluckoff** [reply]
Yes of course, here is the link
https://codepen.io/sooqtmrd-the-animator/pen/VwVWVrR

--
I also have a problem with displaying cells in the row below left a comment. Maybe you can help me  :)
